### PR TITLE
Fix body parser bug

### PR DESCRIPTION
**Pull request by thisdapo**
- [x] Parse request body correctly.

### CODE REVIEW

File: extract-url.js [hunk 1]
```diff
@@ -27,9 +27,8 @@ const searchForKeywords = async (url ) => {
       try {
       await driver.get(`${url}?search=${keyword.split(" ").join('+')}`);
```
Comment (JosuaKrause) [inline extract-url.js:28]: is there any effect to this line? it looks like it discards the result and if an error happens it gets silently ignored

**Review by JosuaKrause — CHANGES_REQUESTED** [review]
dependencies changed but package-lock was not updated.

also, see comments

**Comment by thisdapo** [comment]
> dependencies changed but package-lock was not updated.
> 
> also, see comments

pushed...

**Review by JosuaKrause — APPROVED** [review]
still not sure about the driver.get line but lgtm

**Pull request merged**